blenderproc/__init__.py

```diff
@@ -34,17 +34,19 @@
     import traceback
     # extract the basename of the file, which is the first in the traceback
     stack_summary = traceback.extract_stack()
-    file_names_of_stack = [os.path.basename(file_summary.filename) for file_summary in stack_summary]  
+    file_names_of_stack = [os.path.basename(file_summary.filename) for file_summary in stack_summary]
+    # check if blenderproc is called via python3 -m blenderproc ...
+    is_module_call = file_names_of_stack[0] == "runpy.py"
     if sys.platform == "win32":
         is_bproc_shell_called = file_names_of_stack[2] in ["metadata.py", "__main__.py"]
         is_command_line_script_called = file_names_of_stack[0] == "command_line.py"
         
-        is_correct_startup_command = is_bproc_shell_called or is_command_line_script_called
+        is_correct_startup_command = is_bproc_shell_called or is_command_line_script_called or is_module_call
```

Comment:
Did you check this on windows?

Reply:
yes, I tested a simple `python -m blenderproc...` command and it worked after applying the changes